### PR TITLE
134 feature/hide show option dashboard widget

### PR DESCRIPTION
**Pull request by rajanit2000**

### CODE REVIEW

File: include/Core/UI/Setting/CoreSetting.php [hunk 1]
```diff
@@ -184,4 +186,30 @@ protected function get_user_roles( $option ) {
...
 	}
+
+	/**
+	 * Renders the Email Log `Remove Data on Uninstall?` settings.
+	 *
+	 * @param array $args
+	 */
+	public function render_dashboard_widget_status_settings( $args ) {
+		$option      = $this->get_value();
+		$remove_data = $option[ $args['id'] ];
```
Comment (sudar) [inline include/Core/UI/Setting/CoreSetting.php]: This variable name should be `hide_dashboard_widget`.

This looks like a copy/paste mistake. When you copy/paste code, make sure you make all the necessary changes.

File: include/Core/UI/Component/DashboardWidget.php [hunk 2]
```diff
@@ -24,11 +24,18 @@ public function load() {
...
 	 */
 	public function register() {
-		wp_add_dashboard_widget(
-			'email_log_dashboard_widget',
-			__( 'Email Logs Summary', 'email-log' ),
-			array( $this, 'render' )
-		);
+		$dashboard_status = false;
+		$options          = get_option( 'email-log-core' );
+		if( isset($options['dashboard_widget_status']) )
```
Comment (sudar) [inline include/Core/UI/Component/DashboardWidget.php]: Move this condition here https://github.com/sudar/email-log/blob/dev/2.3.0/include/Core/UI/UILoader.php#L58

Also make sure every if statement has `{` even if it is just one statement.

**Comment by rajanit2000** [comment]
@sudar i have done this review changes

File: include/Core/UI/UILoader.php [hunk 3]
```diff
@@ -54,10 +54,17 @@ public function load() {
 	 */
 	protected function initialize_components() {
 		$this->components['core_settings'] = new Setting\CoreSetting();
+		$dashboard_status                  = false;
```
Comment (sudar) [inline include/Core/UI/UILoader.php]: Move this logic to a seperate function.

Comment (sudar) [reply include/Core/UI/UILoader.php]: And fix all WordPress coding standard issues.

**Comment by rajanit2000** [comment]
@sudar i have done this changes

**Pull request merged**